library/dokku_ps_scale.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.dokku_utils import subprocess_check_output
+import subprocess
+import re
+
+DOCUMENTATION = """
+---
+module: dokku_ps_scale
+short_description: Manage process scaling for a given dokku application
+options:
+  app:
+    description:
+      - The name of the app
+    required: True
+    default: null
+    aliases: []
+  scale:
+    description:
+      - A map of scale values where proctype => qty
+    required: True
+    default: {}
+    aliases: []
+  skip_deploy:
+    description:
+      - Whether to skip the corresponding deploy or not. If the task is idempotent
+        then leaving skip_deploy as false will not trigger a deploy.
+    required: false
+    default: false
+author: Gavin Ballard
+requirements: [ ]
+"""
+
+EXAMPLES = """
+- name: scale web and worker processes
+  dokku_ps_scale:
+    app: hello-world
+    scale:
+      web: 4
+      worker: 4
+
+- name: scale web and worker processes without deploy
+  dokku_ps_scale:
+    app: hello-world
+    skip_deploy: true
+    scale:
+      web: 4
+      worker: 4
+"""
+
+
+def dokku_ps_scale(data):
+    command = "dokku --quiet ps:scale {0}".format(data["app"])
+    output, error = subprocess_check_output(command)
+
+    if error is not None:
+        return output, error
+
+    output = [re.sub(r"\s\s+", "", line) for line in output]
+    scale = {}
+
+    for line in output:
+        if ":" not in line:
+            continue
+        proctype, qty = line.split(":", 1)
+        scale[proctype] = qty
+
+    return scale, error
+
+
+def dokku_ps_scale_set(data):
+    is_error = True
+    has_changed = False
+    meta = {"present": False}
+
+    values = []
```

Comment:
Please give this a more specific name, e.g. scale_list / procs_to_scale (or similar)